astropy/cosmology/connect.py

```diff
@@ -13,6 +25,11 @@
 __doctest_skip__ = __all__
 
 
+# NOTE: private b/c RTD error
+_CosmoT = TypeVar("_CosmoT", bound="Cosmology")
+_MT = TypeVar("_MT", bound="Mapping")  # type: ignore[type-arg]
```

Comment:
Does the documentation build succeed if these are defined in the `if TYPE_CHECKING:` block?

Reply:
A good suggestion!

Reply:
It worked locally. But apparently RTD is different.

Reply:
reverting.

Reply:
Unfortunately this didn't work. Resolving.

Reply:
I want to have a look at the RTD logs to see if I can figure out what the problem might have been, but if I can't find a solution quickly then we should squash and merge.